src/Widgets/DataConditionWidget/DataConditionWidgetClass.ts

```diff
@@ -0,0 +1,9 @@
+import { provideWidgetClass } from 'scrivito'
+
+export const DataConditionWidget = provideWidgetClass('DataConditionWidget', {
```

Comment:
Please extract (or remove) `DataConditionWidget` from this PR, since it is no longer in use.